include/radix/renderer/Renderer.hpp

```diff
@@ -113,7 +113,7 @@ class Renderer {
     bool uniformBuffers;
   } support;
 
-  uint lightsUBO;
+  unsigned lightsUBO;
```

Comment:
there is still one left :)
i thought we already replaced them with "unsigned int"